CRITICAL - 2022-03-31 02:25:04 -->  route cannot be found while reverse-routing.
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\HTTP\RedirectResponse.php(56): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('')
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Common.php(832): CodeIgniter\HTTP\RedirectResponse->route(false)
#2 C:\xampp\htdocs\web2\app\Controllers\Register.php(57): redirect('/register')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2022-03-31 02:27:51 --> md5(): Passing null to parameter #1 ($string) of type string is deprecated
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8192, 'md5(): Passing ...', 'C:\\xampp\\htdocs...', 63)
#1 C:\xampp\htdocs\web2\app\Controllers\Register.php(63): md5(NULL, '2y')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
CRITICAL - 2022-03-31 02:32:01 --> password_verify(): Passing null to parameter #1 ($password) of type string is deprecated
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8192, 'password_verify...', 'C:\\xampp\\htdocs...', 63)
#1 C:\xampp\htdocs\web2\app\Controllers\Register.php(63): password_verify(NULL, '2y')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#7 {main}
ERROR - 2022-03-31 02:32:21 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-03-31 02:32:21 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 02:33:17 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-03-31 02:33:17 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(70): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 02:35:53 --> Unknown column 'created_at' in 'field list'
CRITICAL - 2022-03-31 02:35:53 --> Unknown column 'created_at' in 'field list'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(70): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 02:37:33 --> Unknown column 'updated_at' in 'field list'
CRITICAL - 2022-03-31 02:37:33 --> Unknown column 'updated_at' in 'field list'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(70): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:02:23 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:02:23 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(71): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:03:00 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:03:00 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(70): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:03:31 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:03:31 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:04:03 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:04:03 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:05:00 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:05:00 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:05:04 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:05:04 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:05:30 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:05:30 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
ERROR - 2022-03-31 03:06:00 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:06:00 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-31 03:07:25 --> Undefined property: stdClass::$name
#0 C:\xampp\htdocs\web2\app\Controllers\Login.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 26)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->process()
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2022-03-31 03:14:04 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ':0:
AND `username` = :username:
 LIMIT 1' at line 3
CRITICAL - 2022-03-31 03:14:04 --> You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near ':0:
AND `username` = :username:
 LIMIT 1' at line 3
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\web2\app\Controllers\Login.php(22): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->process()
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2022-03-31 03:15:34 --> Duplicate entry '' for key 'PRIMARY'
CRITICAL - 2022-03-31 03:15:34 --> Duplicate entry '' for key 'PRIMARY'
#0 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tb...', 0)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(670): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\BaseModel.php(738): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Model.php(625): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\web2\app\Controllers\Register.php(69): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Register->process()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Register))
#10 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2022-03-31 03:17:37 --> Undefined property: App\Controllers\Login::$session
#0 C:\xampp\htdocs\web2\app\Controllers\Login.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 30)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->process()
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-03-31 03:25:49 --> Undefined property: stdClass::$level
#0 C:\xampp\htdocs\web2\app\Controllers\Login.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 31)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->process()
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-03-31 09:42:25 --> Undefined property: App\Controllers\Login::$session
#0 C:\xampp\htdocs\web2\app\Controllers\Login.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined prope...', 'C:\\xampp\\htdocs...', 33)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\Login->process()
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2022-03-31 10:24:11 --> Undefined array key "nama"
#0 C:\xampp\htdocs\web2\app\Views\admin-super\v_datasuperadmin.php(54): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 54)
#1 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Common.php(1089): CodeIgniter\View\View->render('admin-super/v_d...', Array, true)
#4 C:\xampp\htdocs\web2\app\Controllers\DataUser.php(13): view('admin-super/v_d...', Array)
#5 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(832): App\Controllers\DataUser->index()
#6 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DataUser))
#7 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\web2\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\web2\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(43): require_once('C:\\xampp\\htdocs...')
#10 {main}
